crates/net/src/sim2h_worker.rs

```diff
@@ -113,21 +113,33 @@ impl Sim2hWorker {
         Ok(instance)
     }
 
+    fn backoff(&mut self) {
+        if self.connection_timeout_backoff < MAX_CONNECTION_TIMEOUT_MS {
+            self.inner_set_backoff(self.connection_timeout_backoff*2);
+        }
+    }
+
+    fn inner_set_backoff(&mut self, backoff: u64) {
+        self.connection_timeout_backoff = backoff;
+        debug!(
+            "BACKOFF setting reconnect interval to {}",
+            self.connection_timeout_backoff
+        );
+        self.reconnect_interval = Duration::from_millis(self.connection_timeout_backoff)
+    }
+
+    fn reset_backoff(&mut self) {
+        if self.connection_timeout_backoff > INITIAL_CONNECTION_TIMEOUT_MS {
```

Comment:
@zIppy Can you explain this logic a bit?

Reply:
I wonder if it would make more sense to ensure that the timeout can never be less than the initial. Or is this desirable in some situations?

Reply:
The idea here is simply to only set the backoff if its actually above the initial amount.  Just saves a bit of computation since this gets called every time through the tick when the connection is ready